CONTRIBUTING.md

```diff
@@ -46,16 +46,49 @@ git push fork feature
 
 Open a pull request against the main `opentelemetry-cpp-contrib` repo.
 
-### How to Receive Comments
+### Build and Run Code Examples
+
+TBD
+
+## Testing
+
+TBD
+
+## Contributing Rules
+
+Follow the OpenTelemetry [C++ SDK coding standards](https://github.com/open-telemetry/opentelemetry-cpp?tab=readme-ov-file#supported-c-versions)
+
+Include tests for new features or bug fixes
+
+
+## How to Receive Comments
 
 * If the PR is not ready for review, please put `[WIP]` in the title, tag it
   as `work-in-progress`, or mark it as [`draft`](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
 * Make sure [CLA](https://identity.linuxfoundation.org/projects/cncf) is
   signed and CI is clear.
+  
+## How to Get PRs Merged
+
+Address any reviewer 
+
+Ensure all tests pass
+
+Squash and merge your commit when approved
+
+
+
+## Further Help
+
+The OpenTelemetry C/C++ special interest group (SIG) meets regularly. See the
+OpenTelemetry [community](https://github.com/open-telemetry/community#cc-sdk)
```

Comment:
Please point to the opentelemetry-cpp readme, like below:

https://github.com/open-telemetry/opentelemetry-cpp?tab=readme-ov-file#contributing